### ISSUE TITLE
20200307_09_lviv_vinnitsa

### ISSUE THREAD

**Issue by ABatalov** [scholokov/long-travel-2#4082]
!!! ANY FEATURE CAN BE CHECKED SEVERAL TIMES BY DIFFERENT TESTERS !!!

 List of the ready features:

 - [ ] Filling of the travel page
 - [ ]  Title
 - [ ] Photo that describes the travel
 - [ ] Map on the page of the travel
 - [ ] Photos on the page of the travel
 - [ ] Search function
 - [ ] Map

https://test.long-travel.live/20200307_09_lviv_vinnitsa

**Comment by ABatalov** [comment]
@BohdanaProtsyk, if you started testing, please move this task to testing in progress.

**Comment by BohdanaProtsyk** [comment]
Так, почала тестування. Через технічні проблеми з електроенергією зможу
працювати після 15:00год.

пн, 7 серп. 2023, 11:33 користувач ALEXANDER ***@***.***>
пише:
